pkg/topology/kademlia/binprefix.go

```diff
@@ -0,0 +1,90 @@
+package kademlia
```

Comment:
copyright missing. also, if there are unit tests for this code (hopefully there are specific ones) we could move them to a separate test file too

Reply:
surprisingly, there are no tests that check that the correct prefixes are generated.